napari/layers/base/base.py

```diff
@@ -1304,38 +1226,41 @@ def _world_to_displayed_data_ray(
         vector_data_ndisplay /= np.linalg.norm(vector_data_ndisplay)
         return vector_data_ndisplay
 
-    def _world_to_data_dims_displayed(
-        self, dims_displayed: List[int], ndim_world: int
+    def _world_to_layer_dims(
```

Comment:
I just noticed the following similar function: https://github.com/napari/napari/blob/87a263328fd71c8d3fc6252fe8ee2d09056ae776/napari/layers/utils/layer_utils.py#L657

Maybe we can just use one for this correspondence? Or there a difference I'm not seeing?

Reply:
I think the difference is that the former accounts for the full set of dims, the latter only the displayed dims. IIRC there are some subtleties (i.e. probably illogical slicing behaviour) which means these should be dealt with very carefully - @kevinyamauchi may have some more specific info here

Reply:
I'm going to leave both for now, but am hoping we can simplify the dimension correspondence logic soon.